pod/perlfunc.pod

```diff
@@ -8484,6 +8484,16 @@ combinations to test comprehensively in the time available to it each run.  It
 can test a random subset each time, and should there be a failure, log the seed
 used for that run so that it can later be used to reproduce the same results.
 
+The C<PERL_RAND_SEED> environment var may be used to cause C<srand()> called
+without arguments to return a consistent seed each time it is called (whether
+called explicitly or implicitly). The exact seeding is unspecified, but
+using different values for C<PERL_RAND_SEED> should produce different
+results. This is intended for debugging and is only guaranteed to produce
+consistent results between invocations of the same perl executable. Note
+that this env variable is read once during process startup, changing it
+during the program flow will not affect the currently running process.
+See L<perlrun> for more details.
```

Comment:
Should this mention that the value of `PERL_RAND_SEED` needs to be a non-negative integer?  (the perlrun change does mention it)

Also "environment var", "env variable", should these be spelled in full (or be consistent)?

Reply:
I guess since you bring it up then yes. :-) Ill make both changes later today.